src/samples/article-history/v2/prc.json

```diff
@@ -33,6 +33,7 @@
           "version": 1,
           "type": "research-article",
           "doi": "10.7554/eLife.1234567890",
+          "versionDoi": "10.7554/eLife.1234567890.4",
```

Comment:
> I've gone back to the XML: https://github.com/elifesciences/XML-mapping/blob/bc20d5db27598b8529288ef8a1573350a2a0ebee/elife-1234567890-v2/elife-1234567890-v2.xml#L21-L23
> 
> And to what JATS have to say about `article-id` and `specific-use`:
> 
> * https://jats.nlm.nih.gov/publishing/tag-library/1.1d2/element/article-id.html
> * https://jats.nlm.nih.gov/publishing/tag-library/1.1d2/attribute/specific-use.html
> 
> And I think `doiVersion` would actually be best here. There wouldn't be any association with epp/preprints and by putting the subject (`doi`) first it makes the `Version` portion more obvious it's qualifying `doi`, rather than qualifying `version` .
> 
> Does anybody have any objections to that? I'm happy to make the updates and apologise for being a nuisance.

Happy to switch to `versionDoi` in the schema. I will do the refactor, if I get a chance, today. We will adjust in our api-dummy PR.